components/post_view/floating_timestamp/floating_timestamp.jsx

```diff
@@ -38,18 +40,18 @@ export default class FloatingTimestamp extends React.PureComponent {
         );
 
         let className = 'post-list__timestamp';
-        if (this.props.isScrolling) {
+        if (isScrolling) {
             className += ' scrolling';
         }
 
-        if (this.props.isRhsPost) {
+        if (isRhsPost) {
             className += ' rhs';
         }
 
         return (
             <div
                 className={className}
-                style={this.props.stylesOverride}
+                style={toastPresent ? {top: '50px'} : null}
```

Comment:
Why don’t we  use classname for this? We are already adding classes depending on what is around the floating timestamp, so another if wouldn’t hurt and it would probably be cleaner.

Reply:
Classes already exist and there weren't part of the PR. As i said i am not a fan personally to use classes in out codebase anymore for things which are not reusable, or at least applied unconditionally.  Things like these tend to make maintenance of css hard. There were some arguments in the past about performance of css vs inline styles in react but with the level of nesting we normally create the browsers will have a much harder time figuring out the specificity IMO.

In this case i am gonna change to class as it is popular behaviour and i am like 1/5 on this.